examples/dispatch-custom-event/README.md

```diff
@@ -0,0 +1,32 @@
+# Fire a Custom Event
+
+**Test Code**: [index.js](index.js)
+
+**Tested Page**: [index.html](index.html)
+
+This example shows how to fire [events](https://developer.mozilla.org/en-US/docs/Web/API/Event) on the webpage with TestCafe.
+
+During the test the `t.dispatchEvent` fires a `touchstart` event on the target.
+
+When a [touchstart event](https://developer.mozilla.org/en-US/docs/Web/Events/touchstart) is triggered on a target item, the page dislays the event's information:
+
+* event's constructor
+* whether the event is [cancelable](https://developer.mozilla.org/en-US/docs/Web/API/Event/cancelable)
+* whether the event [bubbles](https://developer.mozilla.org/en-US/docs/Web/API/Event/bubbles)  
+
+TestCafe invokes the [CustomEvent Constructor](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent) for any event that is neither a [Mouse](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/dispatchevent.html#mouse-events), a [Keyboard](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/dispatchevent.html#keyboard-events), an [Input](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/dispatchevent.html#input-events), a [Focus](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/dispatchevent.html#focus-events) nor a [Pointer](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/dispatchevent.html#pointer-events) event.
+
+To tell TestCafe to use the [TouchEvent Constructor](https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent/TouchEvent), the `eventConstructor` property is passed to the method's `options`.
+
+When `touchstart` fires, the `t.expect.eql` method checks the values displayed on the page.
```

Comment:
why are you saying this and what do you *want* to say? can we just remove this sentence and maybe turn it into a comment? I haven't yet read the source code, and most readers that get to this point will not have read it either. We should either provide some context to make it meaningful or remove it altogether.

```suggestion
```